todo_app/tests/app_test.py

```diff
@@ -0,0 +1,113 @@
+    requests_mock.get('https://api.trello.com/1/search', text=board_data)
+    requests_mock.get('https://api.trello.com/1/boards/5f58c9c553d5e50e89e24311/lists', text=listinboard_data)
+    requests_mock.get('https://api.trello.com/1/boards/5f58c9c553d5e50e89e24311/cards', text=cardsinboard_data)
+    
+    assert 'Book my holiday' in client.get('/').data.decode("utf-8") 
```

Comment:
This test doesn't work as the flask session is not active during the lifecycle of this test